src/hash_to_curve/map_scalar.rs

```diff
@@ -0,0 +1,48 @@
+//! Implementation of hash-to-field for Scalar values
+
+use super::HashToField;
+use crate::generic_array::{typenum::U48, GenericArray};
+use crate::scalar::Scalar;
+
+impl HashToField for Scalar {
```

Comment:
This is actually used in implementing KeyGen from the bls-signatures spec. It's not strictly necessary here, but nice for implementors that would like to use the HashToField trait generically. I'm happy to remove it.

Reply:
Now that `HashToField` has been reworked to not be derived on curve points, I'm okay with this staying around. It would be necessary in any case if someone wanted to implement hash-to-curve for Jubjub.